subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/fixture/GradleBuildExperimentRunner.java

```diff
@@ -123,6 +124,9 @@ public void doRun(BuildExperimentSpec experiment, MeasuredOperationList results)
             } catch (IOException e) {
                 e.printStackTrace();
             }
+            if (connector != null) {
+                connector.disconnect();
+            }
```

Comment:
Why not ~~close~~ disconnect the connector in the same block where it's created?

Reply:
Because Gradle profiler also uses a connector and both use the same `ConnectorServices`. I only want to do the disconnect aka "this process won't be using the TAPI for now" after the test.

The cleanup of the connector should probably go to Gradle profiler instead of doing it here as part of our performance tests. I'll see if the tests still work on Windows when doing `connector.disconnect()` in the same block we use the connector.

Reply:
This would be great to mention here in a comment for posterity.

Reply:
I moved the call to disconnect to the creation of `ConnectorServices` and there are no file locking problems. Let's leave it like that.